build.gradle

```diff
@@ -43,15 +12,15 @@ buildscript {
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:7.4.2"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$orgJetbrainsKotlin"
-        classpath "org.jlleitschuh.gradle:ktlint-gradle:11.1.0"
+        classpath libs.android.gradlePlugin
+        classpath libs.kotlin.gradlePlugin
+        classpath libs.ktlint.gradlePlugin
     }
 }
 
 plugins {
```

Comment:
Since we are using version 7.4.2 of Gradle, I think we need to use an annotation here as the Android documentation says:

> Note: If you are using a version of Gradle below 8.1, you need to annotate the plugins{} block with @Suppress("DSL_SCOPE_VIOLATION") when using version catalogs. Refer to [issue #22797](https://github.com/gradle/gradle/issues/22797) for more info.

Reply:
We are not using kotlin dsl yet. We would need to do that in case we had `build.gradle.kts` files. It's not the case and there is no error shown in the IDE either (or at least I cannot see it), so I think there is nothing to do here.